src/test/java/core/basesyntax/service/impl/ReportCreatorServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreatorService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportCreatorServiceImplTest {
+    public static final String FIRST_KEY = "dummy1";
+    public static final String SECOND_KEY = "dummy2";
+    private static ReportCreatorService reportCreatorService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportCreatorService = new ReportCreatorServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.fruitStorage.put(FIRST_KEY, 3);
+        Storage.fruitStorage.put(SECOND_KEY,0);
```

Comment:
use autoformatter everywhere
```suggestion
        Storage.fruitStorage.put(SECOND_KEY, 0);
```

Reply:
Done